### PR TITLE
feature: include number of communities in the reputation calculation

### PR DESCRIPTION
**Pull request by lucasmenendez**
the different scores quotas updated to:
```
maxFollowersReputation = 10
maxElectionsReputation = 10
maxVotesReputation     = 25
maxCastedReputation    = 45
maxCommunityReputation = 10
maxReputation          = 100
```

### CODE REVIEW

File: mongo/mongo.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func New(url, database string) (*MongoStorage, error) {
 	ms.authentications = client.Database(database).Collection("authentications")
 	ms.notifications = client.Database(database).Collection("notifications")
 	ms.userAccessProfiles = client.Database(database).Collection("userAccessProfiles")
-	ms.communitites = client.Database(database).Collection("communities")
+	ms.communities = client.Database(database).Collection("communities")
```
Comment (p4u) [inline mongo/mongo.go:107]: This breaks everything @lucasmenendez 
We should think about how to push this fix...

**Review by p4u — CHANGES_REQUESTED** [review]

File: mongo/mongo.go (hunk 1 above)
Comment (p4u) [reply mongo/mongo.go:107]: Because you are changing the collection name, and this has direct impact to the database. The collection will be empty.

Comment (lucasmenendez) [reply mongo/mongo.go:107]: But only changes the name of the struct attribute, the collection name remains the same.

Comment (elboletaire) [reply mongo/mongo.go:108]: The collection always had the correct `communities` name, it's just the struct property.

**Review by p4u — APPROVED** [review]

**Pull request merged**